### DISCUSSION TITLE
视频地址能不能以多个ip的形式返回呢？

### DISCUSSION THREAD

**Discussion by luguoguo** [Mrs4s/go-cqhttp#962] [Ideas]
我使用[vscode-qq](https://github.com/takayama-lily/vscode-qq)时，看到自己加的群里发的视频是以多ip的形式返回的，例如：

> http://58.217.244.48/,http://180.97.118.173:80/,http://175.6.85.121:80/,http://14.116.137.89:80/qqdownload?ver=537065138&rkey=3081fd0201010481f53081f202010102010002045a8e43860481a633303531303230313030303433363330333430323031303036323034316432656536343030323033376131616668903230343638646133313361303230343630663164346538303431303336613261623334303062346133656535663165633131613462623466643237313230333761316462393032303130303034313430303030303031383636363936633635373437393730363310303030303030343331333033303333020460f1d4eb04350000000866696c657479706500000004313030330000000b646f776f656e63727970740000000130000000147175696300000001300400&filetype=1003&videotype=1&subvideotype=0&term=unknow

有时候一个地址获取不到会返回400，而我通过vscode-qq的多个ip不断尝试，在返回400非常高的时段也有几率能获取到视频。

**Comment by takayama-lily** [comment]
这是bug！应该已经修了

**Reply by takayama-lily** [reply]
我指vscqq的bug
